zephyr/Kconfig

```diff
@@ -118,6 +118,15 @@ config POSIX_MAX_FDS
     # z_wpa_event_sock - 1 socketpair = 2
     default 7 if !POSIX_API
 
+config BSS_MAX_IDLE_TIME
+    int "BSS max idle timeout in seconds"
+    range 1 64000
```

Comment:
how do we disable this? Shouldn't we have 0 included?

Reply:
Agree.